motioneye/handlers.py

```diff
@@ -1338,28 +1340,78 @@ def download(self, camera_id, filename):
                 'filename': filename, 'id': camera_id})
         
         camera_config = config.get_camera(camera_id)
+
+        # To facilitiate cross-browser HTML5 <video> playback we need
+        # to support HTTP Range requests.
+        # (The below adapted from Tornado's StaticFileHandler)
+        request_range = None
+        range_header = self.request.headers.get("Range")
+        if range_header:
+            request_range = httputil._parse_request_range(range_header)
+
         if utils.local_motion_camera(camera_config):
-            content = mediafiles.get_media_content(camera_config, filename, 'movie')
-            
+            full_path = os.path.join(camera_config.get('target_dir'), filename)
+            size = os.stat(full_path).st_size
+            if request_range:
```

Comment:
I think we can remove it now. The video will be unseekable until fully downloaded, but we will release it faster and can handle this in next PR. It still be much usable then now.